catalog/CSIC/CSIC_SPEI_2_8.jsonnet

```diff
@@ -0,0 +1,330 @@
+        name: 'SPEI_44_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 44 months',
+      },
+      {
+        name: 'SPEI_45_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 45 months',
+      },
+      {
+        name: 'SPEI_46_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 46 months',
+      },
+      {
+        name: 'SPEI_47_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 47 months',
+      },
+      {
+        name: 'SPEI_48_month',
+        description: 'Standardized Precipitation-Evapotranspiration Index (SPEI) where precipitation and evapotranspiration data was accumulated over the previous 48 months',
+      },
+    ],
+    'gee:visualizations': [
+      {
+        display_name: 'SPEI',
+        lookat: {
+          lon: 0,
+          lat: 18,
+          zoom: 2,
+        },
+        image_visualization: {
+          band_vis: {
+            min: [
+              -2.33,
+            ],
+            max: [
+              2.33,
+            ],
+            palette: [
+              '8b1a1a',
+              'de2929',
+              'f3641d',
+              'fdc404',
+              '9afa94',
+              '03f2fd',
+              '12adf3',
+              '1771de',
+              '00008b',
+            ],
+            bands: [
+              'SPEI_24_month',
+            ],
+          },
+        },
+      },
+    ],
+    SPEI_24_month: {
+      minimum: -2.33,
+      maximum: 2.33,
+      'gee:estimated_range': false,
+    },
+  },
+  'sci:doi': '10.20350/digitalCSIC/15121', 
+  'sci:citation': |||
+    Product: Beguería, Santiago; Vicente Serrano, Sergio M.; Reig-Gracia, Fergus; Latorre Garcés, Borja; 2023; 
+    SPEIbase v.2.8 [Dataset]; DIGITAL.CSIC; Version 2.8.
+    [https://doi.org/10.20350/digitalCSIC/15121](https://doi.org/10.20350/digitalCSIC/15121).
+  |||,
+  'sci:publications': [
+    {
+      citation: |||
+        Related publication: Vicente-Serrano S.M., Beguería S., López-Moreno J.I. (2010): 
+        A Multi-scalar drought index sensitive to global warming: The Standardized Precipitation Evapotranspiration Index – SPEI. 
+        Journal of Climate 23(7), 1696-1718. [https://doi.org/10.1175/2009JCLI2909.1](https://doi.org/10.1175/2009JCLI2909.1)
+      |||,
+      doi: '10.1175/2009JCLI2909.1',
+    },
+  ],
+  'gee:interval': {
+    type: 'cadence',
+    unit: 'month',
+    interval: 1,
+  },
+  'gee:terms_of_use': |||
+    The SPEI database is available under the [CC-BY 4.0 license](https://creativecommons.org/licenses/by/4.0/).
```

Comment:
The text from here down is okay, but it is redundant to the given license and the citation above.